app/views/articles/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<h1>Nytt Blogginnlegg</h1>
+
+<%= form_for :article, url: articles_path do |f| %>
+	<% if @article.errors.any? %>
+		<div id="error_explanation">
+			<h2>
+				<%= pluralize(@article.errors.count, "grunn") %> til at dette ikke ble lagret:
+			</h2>
+			<ul>
+				<%= @article.errors.full_messages.each do |msg| %>
+					<li><%= msg %></li>
+				<% end %>
+			</ul>
+		</div>
+	<% end %>
+	
+	<p>
+		<%= f.label :tittel %><br>
```

Comment:
This `:tittel` needs to be the same as the value it is the label for (`:title`) so that it gets the correct `for` html-attribute. This goes for all the labels.

If you gave it a different symbol here because that's what it's using to display and you wanted it in Norwegian, you can pass the text it should use as a second parameter:

```erb
<%= f.label :title, "Tittel" %><br>
<%= f.text_field :title %>
``` 

But again, if you are using I18n then you can put this in your language file, and the translated name will be used automatically:

```yml
nb:
  activerecord:
    attributes:
      article:
        title: "Tittel"
```

All this was taken from the [documentation for `ActionView::Helpers::FormHelper#label`](http://apidock.com/rails/ActionView/Helpers/FormHelper/label).